tests/integration/api_packages_generic_test.go

```diff
@@ -26,7 +26,7 @@ func TestPackageGeneric(t *testing.T) {
 	user := unittest.AssertExistsAndLoadBean(t, &user_model.User{ID: 2})
 
 	packageName := "te-st_pac.kage"
-	packageVersion := "1.0.3-te st"
+	packageVersion := "1.0.3-test"
```

Comment:
Why this test passed before? 😱

Reply:
reverted, this is an unnecessary change.